blueocean-web/package.json

```diff
@@ -42,15 +42,20 @@
     "window-handle": "1.0.0"
   },
   "jenkinscd": {
-    "extDependencies": [
+    "import": [
       "history",
       "immutable",
-      "react-router",
       "keymirror",
       "react-redux",
       "react-router",
       "redux",
       "redux-thunk"
+    ],
```

Comment:
@sophistifunk wrt your comment on these imports, yes, we could remove these and just bundle all of this stuff inside the main/bootstrap bundle. I did it this way originally thinking it might be better doing multiple concurrent async loads of smaller bundles Vs one load of a monster bundle. A bit unscientific so it might make sense to remove these imports and see if a monster bootstrap bundle works better. However, not planning to do that in this PR.